keyboards/willoucom/keypad/info.json

```
@@ -0,0 +1,72 @@
+{
+    "manufacturer": "Willou.com",
+    "keyboard_name": "willoucom/keypad",
+    "maintainer": "willoucom",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["C2"],
+        "direct": [
+            ["GP0", "GP1", "GP2", "GP3", "GP4"],
+            ["GP5", "GP6", "GP7", "GP8", "GP9"],
+            ["GP10", "GP11", "GP12", "GP13", "GP14"],
+            ["GP15", "GP16", "GP17", "GP18", "GP19"],
+            ["GP20"],
+            ["GP21"],
+            ["GP22", "GP26"],
+            ["GP27"],
+            ["GP28"]
+        ],
+        "rows": ["D1"]
```

1. Keyboard name adjusted to match directory structure
2. "C2" and "D1" not pins for rp2040 controller
```suggestion
    "keyboard_name": "keypad",
    "maintainer": "willoucom",
    "bootloader": "rp2040",
    "diode_direction": "COL2ROW",
    "features": {
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": true,
        "nkro": true
    },
    "matrix_pins": {
        "direct": [
            ["GP0", "GP1", "GP2", "GP3", "GP4"],
            ["GP5", "GP6", "GP7", "GP8", "GP9"],
            ["GP10", "GP11", "GP12", "GP13", "GP14"],
            ["GP15", "GP16", "GP17", "GP18", "GP19"],
            ["GP20"],
            ["GP21"],
            ["GP22", "GP26"],
            ["GP27"],
            ["GP28"]
        ]
```